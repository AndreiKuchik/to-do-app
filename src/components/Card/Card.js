import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import styles from './Card.module.css';

const Card = (props) => (
  <div className={classNames(styles.card, props.className)}>
    {props.children}
  </div>
);

Card.propTypes = {
  className: PropTypes.any,
  children: PropTypes.any,
};

export default Card;
