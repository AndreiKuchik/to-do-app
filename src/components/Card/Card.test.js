import React from 'react';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('Testing Card component', () => {
  it('Matches DOM Snapshot', () => {
    const domTree = renderer.create(<Card />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
