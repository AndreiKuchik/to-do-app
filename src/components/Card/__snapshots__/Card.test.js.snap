// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Card component Matches DOM Snapshot 1`] = `
<div
  className="card"
/>
`;
